android/crt/build.gradle

```diff
@@ -103,7 +103,6 @@ android {
         cmake {
             path "../../CMakeLists.txt"
             buildStagingDirectory "../../target/cmake-build"
-            version "3.10.2"
```

Comment:
Seems to work without an explicit version. If we face any issues, we can revert back to an explicit version.